frontend/packages/dev-console/src/components/health-checks/health-checks-utils.ts

```diff
@@ -0,0 +1,50 @@
+import { ProbeType, HealthChecksProbeType, HealthCheckProbeDataType } from './health-checks-types';
+
+export const getProbeTypeData = () => {
+  const probeTypeData = {
+    httpGet: {
+      scheme: 'HTTP',
+      path: '/',
+      port: '8080',
+      httpHeaders: {},
+    },
+    tcpSocket: {
+      port: '8080',
+    },
+    command: [''],
+  };
+  return probeTypeData;
+};
+
+export const getHealthChecksProbeData = (probe: string): HealthCheckProbeDataType => {
+  const healthChecksProbeData = {
```

Comment:
Your health check probe data contains exact same object for each of the types and you're sending the same data in all the cases. Why do you even need this util function here?